### PR TITLE
Initial profile documentation

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Welcome to the Content Queue!
+
+![Banner](./images/banner.webp)
```
Comment (MichaelKohler) [inline profile/README.md:3]: I just had to 😁

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The Content Queue allows you to manage your social media posts by curating content in a GitHub workflow. Everything is managed through a GitHub Project board and GitHub Actions.
+
+## Common Workflow
+
+### Setup
+
+Check the [template repository](https://github.com/content-queue/template) for ready available workflows and instructions on how to set them up. Add these to a new repository and go through the credentials setup instructions. Set up the project board with the necessary columns as configured in the GitHub Actions workflows. And don't forget to give access to everyone who needs access to the repo.
```
Comment (MichaelKohler) [inline profile/README.md]: Instructions will be done next :)

File: profile/README.md (hunk 1 above)
Comment (freaktechnik) [reply profile/README.md:3]: Thanks for the art!

File: profile/README.md (hunk 2 above)
Comment (freaktechnik) [reply profile/README.md]: Do you want to adjust this section for the template repository thing?

Comment (MichaelKohler) [reply profile/README.md]: Currently working on the `README` for the template repo which will have all of that info, but we could be a bit more clear on this. Maybe we could even remove the following completely as that would be part of the instructions in the template repo anyway?

> Add these to a new repository and go through the credentials setup instructions. Set up the project board with the necessary columns as configured in the GitHub Actions workflows.

Comment (MichaelKohler) [reply profile/README.md]: Adjusted.

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+![Banner](./images/banner.webp)
+
+The Content Queue allows you to manage your social media posts by curating content in a GitHub workflow. Everything is managed through a GitHub Project board and GitHub Actions.
+
+## Common Workflow
+
+### Setup
+
+Check the [template repository](https://github.com/content-queue/template) for ready available workflows and instructions on how to set them up for yourself.
+And don't forget to give access to everyone who needs access to the repo.
```
Comment (freaktechnik) [inline profile/README.md]: > And don't forget to give access to everyone that should help manage your social media using the content queue.

Or similar?

**Review by freaktechnik — APPROVED** [review]

**Pull request merged**